src/tomocupy/config.py

```diff
@@ -242,6 +242,14 @@ def default_parameter(func, param):
         'default': 0,
         'type': float,
         'help': "Regularization parameter"},
+    'delta-beta': {
```

Comment:
could you call them as retrieve-phase-delta-beta and retrieve-phase-W? just to immediately understand that these pars  are needed for phase retrieval and can be easily found?

Reply:
Done